lib/metrics.js

```diff
@@ -123,7 +129,12 @@ function toMetricsObject(obj) {
         const result = {};
         for (const [key,value] of obj) {
             if (typeof key === 'string') {
-                result[key] = truncateString(value);
+                // increase max string length for error metrics
+                if (ERROR_METRIC_NAMES.indexOf(key) > -1) {
```

Comment:
suggestion: could also use `Array.includes()`
 https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes

Reply:
Good catch - I updated the code to use `Array.includes()`.

Taylor